src/lib/mongoBackend/mongoUpdateContextSubscription.cpp

```diff
@@ -382,6 +382,7 @@ HttpStatusCode mongoUpdateContextSubscription
 
     //
     // Update sub-cache
+    // FIXME PR: this is safe without sem?
```

Comment:
@kzangeli pay special attention to this. Some lines below the cache is accesses without sem take.

Maybe we have talked about this in the past (I have a kind of deja-vu... :). Maybe it is safe to to in this way, but al least a comment should be added about it.

(Similar code in the new mongoUpdateSubscription() function)